aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/Translator.java

```diff
@@ -27,6 +27,9 @@ public static CreateCustomDbEngineVersionRequest createCustomDbEngineVersionRequ
                 .description(model.getDescription())
                 .engine(model.getEngine())
                 .engineVersion(model.getEngineVersion())
+                .sourceCustomDbEngineVersionIdentifier(model.getSourceCustomDBEngineVersionIdentifier())
+                .useAwsProvidedLatestImage(model.getUseAwsProvidedLatestImage())
+                .imageId(model.getImageId())
```

Comment:
These attributes are returned in the DescribeDBEngine response. You should also include them in the `translateFromSdk` method in the Translator. This way, CFN will be aware that they have been applied.